<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1158</width>
    <height>760</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ChoseongGame</string>
  </property>
  <widget class="QPushButton" name="Choseong_1">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>110</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Choseong_2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>110</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Choseong_3">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>110</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Choseong_4">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>260</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Choseong_5">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>260</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Choseong_6">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>260</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Choseong_7">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>410</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Choseong_8">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>410</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Choseong_9">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>410</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Choseong_10">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>110</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Choseong_11">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>260</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Choseong_12">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>410</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Choseong_13">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>560</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Choseong_14">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>560</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Choseong_15">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>560</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Choseong_16">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>560</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextEdit" name="TextInput">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>690</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="acceptRichText">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>847</x>
     <y>0</y>
     <width>31</width>
     <height>761</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QProgressBar" name="TimeLeft">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>831</width>
     <height>23</height>
    </rect>
   </property>
   <property name="maximum">
    <number>300</number>
   </property>
   <property name="value">
    <number>300</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="format">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="OptionButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>700</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>wheel.png</normaloff>wheel.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="RuleButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>700</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>Rule.png</normaloff>Rule.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="StartButton">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>670</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>Play.png</normaloff>Play.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>10</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔고딕</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Best Score:</string>
   </property>
  </widget>
  <widget class="QLabel" name="BScore">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>10</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔고딕</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>None</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="TimeOut">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>300</y>
     <width>481</width>
     <height>111</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ToolTipText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ToolTipText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>120</red>
         <green>120</green>
         <blue>120</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>120</red>
         <green>120</green>
         <blue>120</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>120</red>
         <green>120</green>
         <blue>120</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ToolTipText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>나눔고딕</family>
     <pointsize>76</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Time Out!</string>
   </property>
  </widget>
  <widget class="QLabel" name="NoJaeum">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>660</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔고딕</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>자음이 부족합니다.</string>
   </property>
  </widget>
  <widget class="QLabel" name="NoNatmal">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>660</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔고딕</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>없는 명사입니다.</string>
   </property>
  </widget>
  <widget class="QLabel" name="ListScore">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>70</y>
     <width>71</width>
     <height>561</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔고딕</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="ListWord">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>70</y>
     <width>141</width>
     <height>561</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔고딕</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="NoJungbok">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>660</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔고딕</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>이미 쓴 단어입니다.</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>RuleButton</sender>
   <signal>pressed()</signal>
   <receiver>Dialog</receiver>
   <slot>RuleButton_Pushed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>103</x>
     <y>716</y>
    </hint>
    <hint type="destinationlabel">
     <x>106</x>
     <y>484</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>OptionButton</sender>
   <signal>pressed()</signal>
   <receiver>Dialog</receiver>
   <slot>OptionButton_Pushed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>36</x>
     <y>700</y>
    </hint>
    <hint type="destinationlabel">
     <x>40</x>
     <y>489</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>StartButton</sender>
   <signal>pressed()</signal>
   <receiver>Dialog</receiver>
   <slot>StartButton_Pushed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>808</x>
     <y>683</y>
    </hint>
    <hint type="destinationlabel">
     <x>800</x>
     <y>456</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>TextInput</sender>
   <signal>textChanged()</signal>
   <receiver>Dialog</receiver>
   <slot>TextInput_Changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1007</x>
     <y>701</y>
    </hint>
    <hint type="destinationlabel">
     <x>1007</x>
     <y>539</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>RuleButton_Pushed()</slot>
  <slot>OptionButton_Pushed()</slot>
  <slot>StartButton_Pushed()</slot>
  <slot>TextInput_Changed()</slot>
 </slots>
</ui>
